index.js

```diff
@@ -25,18 +25,18 @@ module.exports = function (options) {
 
     var formatter = new Formatter(options);
 
-    function specFailureDetails (result, specIndex) {
+    function print (message) {
```

Comment:
Good idea to refactor that method, but I think we should put it in formatter and rename it printLine, to go with print and printNewLine. What do you think?